include/sundials/sundials_timestepheuristics.h

```diff
@@ -0,0 +1,286 @@
+/* -----------------------------------------------------------------
+ * Programmer(s): Daniel R. Reynolds @ SMU
+ * -----------------------------------------------------------------
+ * SUNDIALS Copyright Start
+ * Copyright (c) 2002-2023, Lawrence Livermore National Security
+ * and Southern Methodist University.
+ * All rights reserved.
+ *
+ * See the top-level LICENSE and NOTICE files for details.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ * SUNDIALS Copyright End
+ * -----------------------------------------------------------------
+ * SUNDIALS time step heuristics class. These objects constrain step
+ * sizes based on solver heuristics.
+ * ----------------------------------------------------------------*/
+
+#ifndef _SUNDIALS_TIMESTEPHEURISTICS_H
+#define _SUNDIALS_TIMESTEPHEURISTICS_H
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sundials/sundials_context.h>
+#include "sundials/sundials_types.h"
+#include <sundials/sundials_nvector.h>
+
+#ifdef __cplusplus /* wrapper to enable C++ usage */
+extern "C" {
+#endif
+
+/* -----------------------------------------------------------------
+ * SUNTimestepHeuristics types (currently, only "H" is implemented; others
+ * are planned):
+ *    STD     - standard heuristic stepsize constraints
+ *    NULL    - placeholder object that applies no constraints
+ * ----------------------------------------------------------------- */
+
+typedef enum
+{
+  SUN_TIMESTEPHEURISTICS_STD,
```

Comment:
Do we prefer to call them standard, or default? I think this should match how we talk about them in the docs.

Reply:
Looks like the implementation is called "default", so we should change this enum name to "default".